pilot/api/data.py

```diff
@@ -47,6 +49,51 @@ def transfer(self, files):
             raise Exception('Files dictionary does not conform: scope, name, destination')
 
 
+class StageOutClient(object):
+
+    def __init__(self, site=None):
+        super(StageOutClient, self).__init__()
+
+        # Check validity of specified site
+        self.site = os.environ.get('VO_ATLAS_AGIS_SITE', site)
```

Comment:
Doesn't look good with an ATLAS hardcoding - is anyone ever setting VO_ATLAS_AGIS_SITE? I don't see it in any logs. Do we need it?